docs/upgrading.md

```diff
@@ -71,10 +71,25 @@ Change the Contour image version to `docker.io/projectcontour/contour:v1.0.0-bet
 The recommended version of Envoy remains unchanged from Contour 0.15.
 Ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.11.1`.
 
+### Split deployment/daemonset now the default
+
+We have changed the example installation to use a separate pod installation, where Contour is in a Deployment and Envoy is in a Daemonset.
+Separated pod installations separate the lifecyle of Contour and Envoy, increasing operability.
+Because of this, we are marking the single pod install type as officially deprecated.
+If you are still running a single pod install type, please review the [`contour` example](../examples/contour/README.md) and either adapt it or use it directly.
+
 ### Leader Election
 
 Contour 1.0.0-beta.1 enables leader election by default.
-No specific configuration is required 
+No specific configuration is required if you are using the example deployment.
+
+Leader election requires that Contour have write access to a ConfigMap called `leader-elect` in the project-contour namespace.
```

Comment:
Could you please add that this is already taken care of by the example rbac rules. include hyperlink to the lines in 01-rbac.yaml